roles/pulp_devel/tasks/install_docs_requirements.yml

```diff
@@ -17,6 +17,18 @@
   environment:
     SETUPTOOLS_USE_DISTUTILS: stdlib
 
+- name: Install plugin requirements for testing locally
+  pip:
+    requirements: '{{ item.value.source_dir }}/functest_requirements.txt'
```

Comment:
Can we safely assume all plugins have such a file? Why is it `functest_requirements.txt` if for pulpcore we install `test_requirements.txt`?

Reply:
`functest_requirements.txt` comes from plugin-template: https://github.com/pulp/plugin_template/tree/master/templates/test
pulpcore combined `functest_requirements.txt` with `unittest_requirements.txt` into `test_requirements.txt`: https://github.com/pulp/pulpcore/blob/master/test_requirements.txt

Reply:
So pulpcore doesn't follow the plugin_template? Or has the move not been ported to pulpcore?
Anyway, I'm fine with this change as long as no one plugin complains.

Reply:
it follows plugin-template by having unit and func requirements, but it was clever than plugin-template and made a shortcut for installing unit and func requirements together